### ISSUE TITLE
the Resolution is lower

### ISSUE THREAD

**Issue by brightendavid** [mjkwon2021/CAT-Net#1]
when I test your code ,I find the **Resolution  of output is lower than the  input**.How can I solve the problem.

**Comment by CauchyComplete** [comment]
Hi,

Heatmaps use a fixed width, but you can use the `pred` variable in `tools/infer.py` to handle the full-resolution output.

https://github.com/mjkwon2021/CAT-Net/blob/17d294d82f9e402484bf82637ba992fdc304c0e7/tools/infer.py#L135

**Comment by CauchyComplete** [comment]
Hi,

Heatmaps use a smaller size output, but you can use the `pred` variable in `tools/infer.py` to handle the full-resolution output.

https://github.com/mjkwon2021/CAT-Net/blob/17d294d82f9e402484bf82637ba992fdc304c0e7/tools/infer.py#L135

**Comment by brightendavid** [comment]
thank  you for  your answer, the resoluton of the output  will always be  _(w//4,h//4)_,is that designed on porpose? or it may be  a problem    result from   my torch  version.

**Comment by CauchyComplete** [comment]
@brightendavid Yes, the outputs have 1/4 resolution. This is due to two stride-2 convolutions at the very beginning, as explained in the paper 😊
